Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
+    
+    NSMutableArray *authors = [NSMutableArray array];
+    
+    if (authorNamesAndAddresses) {
+        [authorNamesAndAddresses enumerateKeysAndObjectsUsingBlock:^(id name, id emailAddress, BOOL *stop) {
+            if ([name isKindOfClass:[NSString class]] && [emailAddress isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:(NSString *)emailAddress];
+                [authors addObject:author];
+            }
+        }];
+    }
+    else if (authorNames) {
+        for (id name in authorNames) {
+            if ([name isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:nil];
+                [authors addObject:(NSString *)author];
+            }
+        }
+    }
+    else if (authorName) {
+        TMAuthor *author = [[TMAuthor alloc] initWithName:authorName emailAddress:nil];
+        [authors addObject:(NSString *)author];
+    }
+    else {
+        return nil;
```

Comment:
Because I want to give the caller some idea as to whether the author included the key screenshots or not. If they get an empty array, it’s because the author put the wrong kind of data in the array, if they get nil, it means it wasn’t included at all.